tests/rag/test_retriever.py

```
@@ -0,0 +1,74 @@
+
+
+@pytest.fixture(autouse=True)
+def _reset_vector_store() -> None:
+    set_vector_store(MockVectorStore())
+
+
+class _RecordingStore(BaseVectorStore):
+    def __init__(self) -> None:
+        self.last_top_k: Optional[int] = None
+
+    def add_documents(self, documents: List[Dict[str, Any]]) -> None:
```

Ruff (0.14.14)

[warning] Unused method argument: `documents`

(ARG002)